src/template/L1PortalExecuteL2Call.sol

```diff
@@ -106,15 +101,22 @@ contract L1PortalExecuteL2Call is SimpleTaskBase {
 
         bool _found;
         uint256 _matches;
-        for (uint256 _i = 0; _i < _actions.length; _i++) {
-            if (_actions[_i].target == portal && _actions[_i].value == 0) {
-                if (keccak256(_actions[_i].arguments) == keccak256(_expected)) {
-                    _found = true;
-                    _matches++;
+        SuperchainAddressRegistry.ChainInfo[] memory chains = superchainAddrRegistry.getChains();
+        for (uint256 i = 0; i < chains.length; i++) {
+            for (uint256 _i = 0; _i < _actions.length; _i++) {
```

Comment:
This is a bit confusing with the naming of `i` and `_i`, also I dont follow why do we loop through the chains?